packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -97,92 +97,20 @@ export type CustomStyleHooksContextValue = {
 export const CustomStyleHooksContext = React.createContext<CustomStyleHooksContextValue | undefined>(undefined);
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27404&commitId=2decb9ec3a7667032c5282c5c20fac40edb13724&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1902 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|1605722| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>